Sources/TokamakCore/MountedViews/MountedElement.swift

```diff
@@ -117,29 +117,41 @@ public class MountedElement<R: Renderer> {
 }
 
 extension TypeInfo {
-  func injectEnvironment(from environmentValues: EnvironmentValues, into element: inout Any) {
+  func injectEnvironment(from environmentValues: EnvironmentValues, into element: inout Any) -> EnvironmentValues {
+    var modifiedEnv = environmentValues
+    // swiftlint:disable force_try
+    // Extract the view from the AnyView for modification
+    if genericTypes.filter({ $0 is EnvironmentModifier.Type }).count > 0 {
+      // Apply Environment changes:
+      if let modifier = try! property(named: "modifier").get(from: element) as? EnvironmentModifier {
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/MountedViews/MountedElement.swift;
  Line: 126;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 101 characters (`line_length`)